<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfo</class>
 <widget class="QWidget" name="UserInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="back_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(240, 240, 240, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ico/unknown);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nick_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>175</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>[未知]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="info_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>100</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string>昵称:

用户ID:

用户类型:

用户HASH:

服务器:

频道:

在线人数:

</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nick_1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>430</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[未知]</string>
   </property>
  </widget>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>105</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[未知]</string>
   </property>
  </widget>
  <widget class="QLabel" name="utype">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>160</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[未知]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hash">
   <property name="geometry">
    <rect>
     <x>369</x>
     <y>210</y>
     <width>430</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>[未知]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="server">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>[未知]</string>
   </property>
  </widget>
  <widget class="QLabel" name="room">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[未知]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="online_num">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>375</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>[未知]</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="back_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>200</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 0), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="back_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ico/schat_back);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>back_0</zorder>
  <zorder>back_1</zorder>
  <zorder>icon</zorder>
  <zorder>nick_0</zorder>
  <zorder>nick_1</zorder>
  <zorder>id</zorder>
  <zorder>utype</zorder>
  <zorder>hash</zorder>
  <zorder>server</zorder>
  <zorder>room</zorder>
  <zorder>online_num</zorder>
  <zorder>back_2</zorder>
  <zorder>back_3</zorder>
  <zorder>info_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
